kedro-datasets/kedro_datasets/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
```

Comment:
This is a cool trick (didn't know it was possible) but can the `conftest.py` file live in `kedro-datasets/tests` instead?

Reply:
[I don't think so.](https://stackoverflow.com/a/62268839)

I also did try it, just to be sure:

```python
(kedro-datasets) deepyaman@Deepyamans-MacBook-Air kedro-datasets % pytest kedro_datasets/dask --doctest-modules 
============================================================================================= test session starts ==============================================================================================
platform darwin -- Python 3.10.9, pytest-7.2.2, pluggy-1.0.0
rootdir: /Users/deepyaman/github/kedro-org/kedro-plugins/kedro-datasets, configfile: pyproject.toml
plugins: mock-1.13.0, requests-mock-1.10.0, xdist-2.2.1, anyio-3.6.2, forked-1.6.0, cov-3.0.0
collected 1 item                                                                                                                                                                                               

kedro_datasets/dask/parquet_dataset.py F                                                                                                                                                                 [100%]

=================================================================================================== FAILURES ===================================================================================================
_________________________________________________________________________ [doctest] kedro_datasets.dask.parquet_dataset.ParquetDataset _________________________________________________________________________
040 
041         >>> import dask.dataframe as dd
042         >>> import pandas as pd
043         >>> from kedro_datasets.dask import ParquetDataset
044         >>> from pandas.testing import assert_frame_equal
045         >>>
046         >>> data = pd.DataFrame({"col1": [1, 2], "col2": [4, 5], "col3": [[5, 6], [7, 8]]})
047         >>> ddf = dd.from_pandas(data, npartitions=2)
048         >>>
049         >>> dataset = ParquetDataset(
UNEXPECTED EXCEPTION: NameError("name 'tmp_path' is not defined")
Traceback (most recent call last):
  File "/opt/miniconda3/envs/kedro-datasets/lib/python3.10/doctest.py", line 1350, in __run
    exec(compile(example.source, filename, "single",
  File "<doctest kedro_datasets.dask.parquet_dataset.ParquetDataset[6]>", line 2, in <module>
NameError: name 'tmp_path' is not defined
/Users/deepyaman/github/kedro-org/kedro-plugins/kedro-datasets/kedro_datasets/dask/parquet_dataset.py:49: UnexpectedException
=========================================================================================== short test summary info ============================================================================================
FAILED kedro_datasets/dask/parquet_dataset.py::kedro_datasets.dask.parquet_dataset.ParquetDataset
============================================================================================== 1 failed in 0.97s ===============================================================================================
```

Reply:
🥹 Thanks for checking. Feels terrible to ship this file to be honest, maybe let's exclude it in `MANIFEST.in`?

Reply:
@merelcht I think because the `contest.py` is not in the tests directory, which wouldn't get packaged.

Reply:
Ahh I see.. I don't mind it that much.

Reply:
I fixed it real quick; @astrojuanlu maybe you can sanity check that it works, since I'm not super familiar working with manifest files.

Reply:
I confirm it works 👍🏽 

I think shipping stuff that is not user-facing but only for development/testing is an antipattern, I see users doing `from kedro_datasets.conftest import *` otherwise...